libs/application/templates/aosh/work-accident-notification/src/fields/Overview/index.tsx

```
@@ -0,0 +1,108 @@
+          }}
+        />
+      </ReviewGroup>
+      <Box paddingY={[0, 2, 4, 6]}>
+        <Accordion>
+          {employees.map((employee, index) => (
+            <AccordionItem
+              id={`${field.id}-accordion-item-${index}`}
+              label={`${formatMessage(overview.labels.employee)} ${
+                employees.length > 1 ? index + 1 : ''
+              } - ${employee.nationalField.name}`}
+            >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)